### ISSUE TITLE
Optimize webpage for mobile

### ISSUE THREAD

**Issue by merzee** [prismofeverything/organism#52]
Playing on mobile phone is my most common way of progressing games of Go or Chess on my phone. Would be great to see for Organism!

**Comment by prismofeverything** [comment]
Agreed! If I tip my phone and zoom out it works for me currently, but not as smooth as it could be. I'll work on it.
